specifyweb/frontend/js_src/css/workbench.css

```diff
@@ -77,6 +77,16 @@
   --accent-color: var(--search-result);
 }
 
+/* 
+ * Override default font properties: required for handsontable 10 and above
+ * See: https://github.com/handsontable/handsontable/pull/8681
+ */
+.handsontable {
+  @apply text-black dark:text-white;
+  font-family: inherit;
+  font-size: inherit;
+}
+
```

Comment:
I don't think tailwind has any classes for font property inherits. I found this: https://github.com/tailwindlabs/tailwindcss/discussions/6815 but not sure if we need that just for two lines?

Reply:
we have customized our tailwind config for several things like this, so it's no big deal.
but also, having these in the CSS file without `@apply` is not an issue